src/main.c

```diff
@@ -154,6 +162,9 @@ int main(void) {
         while (1) {
         }
 
+#if USB_PID == 0x0013  // Metro m0
+    delay(100);
```

Comment:
Does this have to do with attaching debugger?

Reply:
yes/programmer. when programming the metro m0, the SWD programmer can't attach after the bootloader's been put on. this doesn't affect overall functionality, but makes it easier for us to program

Reply:
OK, would be good to add a comment, and also check USB_VID.

Reply:
@dhalbert please add ^^ thx!